styles/style.css

```diff
@@ -295,4 +318,435 @@
     margin-bottom: 10px;
     line-height: 21px;
   }
+
+  .img1,
+  .img2,
+  .img3,
+  .img4,
+  .img_,
+  .skill-img-dots,
+  .contImage,
+  .contImage2 {
+    display: none;
+  }
+}
+
+@media only screen and (min-width: 768px) {

```

Comment:
- [ ] In this screen size (768px), your desktop style looks like in the video below. Please fix the width of the page and apply fixed width that won't require users to scroll the page horizontally. https://clipchamp.com/watch/kznorbkV6Ws